src/routes/transactions/mappers/common/safe-app-info.mapper.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { faker } from '@faker-js/faker';
+import { safeAppBuilder } from '../../../../domain/safe-apps/entities/__tests__/safe-app.builder';
+import { SafeAppsRepository } from '../../../../domain/safe-apps/safe-apps.repository';
+import { multisigTransactionBuilder } from '../../../../domain/safe/entities/__tests__/multisig-transaction.builder';
+import { SafeAppInfo } from '../../entities/safe-app-info.entity';
+import { SafeAppInfoMapper } from './safe-app-info.mapper';
+
+describe('SafeAppInfo mapper (Unit)', () => {
+  const safeAppsRepositoryMock = jest.mocked({
+    getSafeApps: jest.fn(),
+  } as unknown as SafeAppsRepository);
+
+  const mapper = new SafeAppInfoMapper(safeAppsRepositoryMock);
```

Comment:
We can face issues with this in the future if an internal state is introduced in the mapper.

I would build the mapper before each test.

Reply:
I'd say the mappers should be stateless but yes, I agree it's a best practice, thanks! Changed in [ec85300](https://github.com/5afe/safe-client-gateway-nest/pull/236/commits/ec85300efc26bd20ca0b69b3e6606d6336138dd1)